bluetooth/src/androidLibMain/kotlin/BluetoothMonitor.kt

```diff
@@ -50,7 +50,11 @@ actual class BluetoothMonitor internal constructor(
     }
 
     override val isServiceEnabled: Boolean
-        get() = bluetoothAdapter.isEnabled
+        get() = if (bluetoothAdapter == null) {
```

Comment:
Why? It not optional, if so you can even do bluetoothAdapter?.isEnabled

Reply:
if it is `bluetoothAdapter?.isEnabled` when I run android app I get a crash. I tried moving the call to super.method after register receiver, but I get crash anyway.

Reply:
But we check it to be not null inside create() how it's can be possible?

Reply:
Yeah bit puzzeled on this one, @corrado4eyes are you sure this was here? Not the non final abstract field scenario we discusses in the parent class?